salt/modules/rabbitmq.py

```diff
@@ -578,6 +579,29 @@ def list_queues_vhost(vhost, runas=None, *kwargs):
     return res
 
 
+def queue_vhost_exists(queue, vhost, runas=None, *kwargs):
```

Comment:
What's the purpose of putting kwargs into a tuple of positional arguments? I don't see a need for this.